### ISSUE TITLE
Split Screen?

### ISSUE THREAD

**Issue by WebSpectrum** [ExOptsTeam/NFSCExOpts#34]
Does this add split screen play? Thanks

**Comment by ThatD07** [comment]
No, it doesn't. The "split screen fix" just enables that broken feature that was left in the PC version. It doesn't work as intended
